src/components/Browse/Browse.tsx

```diff
@@ -42,8 +53,11 @@ const Browse = () => {
                         dsList.length === 0 ? (
                             <span>No datasets found.</span>
                         ) : (
-
-                            <DataSetList datasets={dsList} />
+                            <div><div className="w3-panel w3-round-large" style={{ display: "inline-block", backgroundColor: "rgba(196,52,34,0.4)", padding: "12px", border: "1px solid rgba(196,52,34,0.5)" }}>
```

Comment:
can you please move the CSS to its file?

Reply:
Addressed.